controller/watertank_controller.go

```diff
@@ -0,0 +1,235 @@
+package controller
```

Comment:
please let the name of this file as waterTankLevels_controller.go and change the smartlight_controller.go to smartLights_controller.go.

In description of this PR ArtesianWell and Hidrometer were not find to PR.

Reply:
File names changed as requested, the ArtesianWell and Hidrometer are already in the PR.